Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV GOOS=linux \
     CGO_ENABLED=1
 
 # this value changes in ./bin/build
```

Comment:
can you also elaborate here on why we use a suffix and not just the word?

Reply:
we have this description in the version.go file... i think that this is enough without adding a description in every reference. let me think if you think otherwise